src/js/modules/accounts/components/forms/EditUserForm.js

```diff
@@ -31,9 +56,13 @@ const renderField = ({
 }) => {
   return (
     <div>
-      <label>{label}</label>
       <div>
-        <input {...input} placeholder={label} type={type} />
+        <input
```

Comment:
I really don't like having this component repeated. You should just put in a separate file. Plus the whole lowerCamelCase sorta pisses me off. Redux form is a mad useful library but their naming sucks